### ISSUE TITLE
Enable scientific+common names view in the taxa search

### ISSUE THREAD

**Issue by kazlauskis** [NERC-CEH/irecord-app#183]
> from Mark Gurney:
Scientific names display when picking from two English names with different taxa. Can they display for all English names?

![screenshot_01](https://user-images.githubusercontent.com/7383162/73547766-85fc0e80-4437-11ea-8eef-afb640aafda0.jpg)

**Comment by sacrevert** [comment]
Agree that this is important and should be prioritised under the next update. The current situation results in some ambiguous offerings. For example, if the user searches "Knapweed", two of the matches have the name "Common Knapweed", but these two link to distinct scientific names/taxonomic concepts.

**Comment by kitenetter** [comment]
Ideally English names should always be displayed for all scientific names as well.

**Comment by sacrevert** [comment]
Yes, absolutely. Ideally there is just one view of any given name with both common and scientific names displayed. This is as per the NPMS/PP app.
